.github/workflows/main.yml

```diff
@@ -1,54 +0,0 @@
-# This is a basic workflow to help you get started with Actions
-
-name: Firely Validation
```

Comment:
Lets hold off removing this one until after JT call with simplifier, as he may be pushing to use Fire.ly Terminal again